remappers/internal/metric.go

```diff
@@ -68,7 +68,7 @@ func AddMetrics(
 			dp.SetStartTimestamp(metric.StartTimestamp)
 		}
 
-		dp.Attributes().PutStr("event.provider", "hostmetrics")
+		dp.Attributes().PutStr("event.module", common.RemapperEventModule)
```

Comment:
The only issue that might happen is if there is a panel/visualization in UI's using event.module for representation since this value is "system" in the System Integration.

Reply:
Hmm, can we confirm this in some way? If not then what do you suggest we should use?

Reply:
I don't see it being used in dashboards, the issue would be for custom system dashboards, if someone has built a dashboard using this field name. 
In that case, having a new field might make sense. 
@tommyers-elastic : WDYT ?

Reply:
Lets go ahead with event.module